app/scripts/utils/__tests__/path-test.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-env mocha */
```

Comment:
Thoughts @diasdavid @RichardLitt is this good, or should we move all tests to a top level `tests` folder?